conduce.py

```diff
@@ -79,6 +90,28 @@ def edit_resource(args):
                     api.update_resource(resource, **vars(args))
 
 
+def edit_entity(args):
+    dataset_id = args.dataset_id
+    entity_id = args.id
+    del vars(args)['dataset_id']
+    del vars(args)['id']
+    entity_history = api.get_entity(dataset_id, entity_id, **vars(args))
+
+    subject = None
+    if args.date is None:
+        subject = entity_history[-1]
+    else:
+        for entity in entity_history:
+            if entity['timestamp_ms'] == util.string_to_timestamp_ms(args.date):
+                subject = entity
+
+    edited_entity = open_in_editor(json.dumps(subject, indent=2))
```

Comment:
if the user types in the wrong date, `subject` could be `None`, so there should be a check, `if subject is None: raise ValueError("{} is an invalid date, use one of the following: {}"...)`

Reply:
Admittedly I did not test the timestamp feature.  It doesn't seem all that useful.